build/Dockerfile

```diff
@@ -17,18 +17,19 @@ RUN pacman --noconfirm -S ghostscript
 RUN pacman --noconfirm -S graphviz
 RUN pacman --noconfirm -S lilypond
 
-# Resilient packages and classes for SILE
-RUN luarocks install resilient.sile
-
 # Curated set of decent fonts for examples
 RUN pacman --noconfirm -S make
 COPY ./build/Makefile-fonts /
 RUN make -f Makefile-fonts allfonts
-# COPY ./build/fonts/OCRB.otf /.fonts
 
-# Should be done
-RUN luarocks list
+# Just to check the version on first run
 RUN sile --version
 
+# Resilient packages and classes for SILE
+# Try to deduce the Lua version used by SILE, which might be different from the system Lua
+RUN v=$(sile --quiet -e 'v = (_VERSION):gsub("Lua ", ""); print(v); os.exit()'); echo $v > /LVER
```

Comment:
```
$sile -q -e 'print(SILE.lua_version); os.exit()'
5.1
```

Reply:
Wow :heart:

Reply:
The Dockerfile shell stuff can be simplified too ;-)

```dockerfile
RUN sile -qe 'print(SILE.lua_version)' > /LVER
```